common/templates/mathjax_include.html

```diff
@@ -5,32 +5,3 @@
 ##
 ## This enables ASCIIMathJAX, and is used by js_textbox
 
-
-%if mathjax_mode is not Undefined and mathjax_mode == 'wiki':
-<script type="text/x-mathjax-config">
-  MathJax.Hub.Config({
-    tex2jax: {inlineMath: [ ['$','$'], ["\\(","\\)"]],
-              displayMath: [ ['$$','$$'], ["\\[","\\]"]]}
-  });
-</script>
-%else:
-<script type="text/x-mathjax-config">
-  MathJax.Hub.Config({
-    tex2jax: {
-      inlineMath: [
-        ["\\(","\\)"],
-        ['[mathjaxinline]','[/mathjaxinline]']
-      ],
-      displayMath: [
-        ["\\[","\\]"],
-        ['[mathjax]','[/mathjax]']
-      ]
-    }
-  });
-</script>
-%endif
-
-<!-- This must appear after all mathjax-config blocks, so it is after the imports from the other templates.
-     It can't be run through static.url because MathJax uses crazy url introspection to do lazy loading of
-     MathJax extension libraries -->
-<script type="text/javascript" src="https://edx-static.s3.amazonaws.com/mathjax-MathJax-727332c/MathJax.js?config=TeX-MML-AM_HTMLorMML-full"></script>
```

Comment:
There's this unprotected use of MathJax `MathJax.Hub.Queue ["Typeset", MathJax.Hub, element[0]]` this will immediately break in [this example](https://github.com/Edraak/edx-platform/blob/edraak/core-changes/common/static/coffee/src/discussion/views/discussion_thread_profile_view.coffee#L13). Didn't follow everything in the site. Closing this as not-mergable.